src/App.js

```diff
@@ -7,26 +7,26 @@ import TeamPage from "./Pages/TeamPage";
 import MatchupPage from "./Pages/MatchupPage";
 import FreeAgentsPage from "./Pages/FreeAgentsPage";
 import LeaguePage from "./Pages/LeaguePage";
+import H2P from './Pages/H2P';
+import NBALiveStats from './Pages/NBALiveStats';
```

Comment:
this is a component, why is it in the pages folder

Reply:
its a page